### ISSUE TITLE
"boot repl" with Clojure >1.9.0-alpha17 and boot >=2.7.0 doesn't start with "Invalid token: ::clojure.test/once-fixtures"

### ISSUE THREAD

**Issue by konstan** [slipstream/SlipStreamServer#1168]
```
java.lang.RuntimeException: Invalid token: ::clojure.test/once-fixtures
clojure.lang.ExceptionInfo: Invalid token: ::clojure.test/once-fixtures
    file: "cider/nrepl/middleware/test.clj"
    line: 106
```

Works only after downgrading to `BOOT_CLOJURE_VERSION=1.9.0-alpha17`.

**Comment by konstan** [comment]
With boot 2.7.2 the error message is missing namespace of the error
```
java.lang.RuntimeException: Invalid token: ::clojure.test/once-fixtures
clojure.lang.ExceptionInfo: Invalid token: ::clojure.test/once-fixtures
    line: 106
```

**Comment by loomis** [comment]
@konstan Can you try adding the dependency (or upgrading if it is there already):
```clojure
[cider/cider-nrepl "0.15.1-SNAPSHOT"]
```
to see if this fixes the issue.

**Comment by konstan** [comment]
@loomis yes, this solves the issue.

**Comment by loomis** [comment]
Including this dependency causes another issue with the classpath:
```
Classpath conflict: org.tcrawley/dynapath version 0.2.3 already loaded, ALSO loading version 0.2.5
```

Probably best to revert to 1.9.0-alpha17 until the build tooling catches up with the fixes.
